app/helpers/application_helper.rb

```diff
@@ -39,9 +39,9 @@ def display_banner
     banner = Banner.last
 
     if !banner.nil? && banner.show
-      content = "<div id='banner-news' class='banner-news'><div class='content'>"
-      content += "#{banner.content}</div></div></br><div class='separator'></div>"
-      return content.html_safe
+      ("<div id='banner-news' class='banner-news'>" +
```

Comment:
i don't love either the content or the interpolated string implementation, but this removes the <br> and makes clearer the 'contents' (haha) of the banner wrapper html.

once we're done with bootstrap and can move the tailwind layout partials outside the tailwind namespace i'd like to remove this method completely and just add the css to the partial. 

(we still need this method for now bc the regular layout and nav partials are calling it)